docs/conf.py

```diff
@@ -0,0 +1,62 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'fairlearn'
+copyright = '2019, Microsoft'
+author = 'Microsoft'
+
+# The full version, including alpha/beta/rc tags
+release = '0.3.0-pre-alpha-interim'
```

Comment:
oh no, another place where we need to keep a consistent version number :-(

Reply:
There may well be a way of reading this from the python